rules/test_host_app/test_host_app.bzl

```diff
@@ -0,0 +1,40 @@
+"""
+Generates test host ios_application targets for each supported version
+
+A consumer of `rules_ios` can optionally load the `generate_test_host_apps` symbol
+and pass in additional attributes to be set in all `ios_application` rules
+"""
+
+load("//rules:app.bzl", "ios_application")
+
+def generate_test_host_apps(**kwargs):
+    versions = [
```

Comment:
wondering this is the opportunity to allow passing of versions list... (and we have a default list of course) I'd imagine the need for doing so if rules_ios doers not keep up

Reply:
Thought about this too, also the ability to set the other `ios_application` attributes (`entitlements`, `families`, etc) but I think we should address use cases as they feel necessary? Otherwise it's easy to make this more generic than necessary IMO

I think we can keep this as is and come back here in the future to make it more customizable only if there's a use case for it